src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -922,6 +922,9 @@ void Schedule::iterateScheduleSection(std::size_t load_start, std::size_t load_e
         return this->snapshots[timeStep].groups.get(groupName);
     }
 
+    const WListManager& Schedule::getWListManager(std::size_t timeStep) const {
```

Comment:
No - this should use `Schedule::operator[]` at call site - trying to get away from the methods which just forward into a snapshot.

Reply:
This is done.